x-pack/plugins/fleet/common/services/file_storage.ts

```diff
@@ -5,32 +5,54 @@
  * 2.0.
  */
 
-import { FILE_STORAGE_DATA_INDEX_PATTERN, FILE_STORAGE_METADATA_INDEX_PATTERN } from '../constants';
+import {
+  FILE_STORAGE_DATA_INDEX_PATTERN,
+  FILE_STORAGE_METADATA_INDEX_PATTERN,
+  FILE_STORAGE_TO_HOST_DATA_INDEX_PATTERN,
+  FILE_STORAGE_TO_HOST_METADATA_INDEX_PATTERN,
+} from '../constants';
 
 /**
  * Returns the index name for File Metadata storage for a given integration
  * @param integrationName
+ * @param forHostDelivery
  */
-export const getFileMetadataIndexName = (integrationName: string): string => {
-  if (FILE_STORAGE_METADATA_INDEX_PATTERN.indexOf('*') !== -1) {
-    return FILE_STORAGE_METADATA_INDEX_PATTERN.replace('*', integrationName);
+export const getFileMetadataIndexName = (
```

Comment:
some unit tests would be nice here

Reply:
Thanks @juliaElastic . I could not find an existing test file as you alluded to above, so created a new one and added tests for these service utilities.